tests/integration-tests/tests/cfn-init/test_cfn_init/test_replace_compute_on_failure/pcluster.config.yaml

```diff
@@ -0,0 +1,30 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Iam:
+    S3Access:
+      - BucketName: {{ bucket_name }}
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  CustomActions:
+    - Script: s3://{{ bucket_name }}/post_install.sh
+Scheduling:
+  {{ scheduler }}:
+    Settings:
+      ScaledownIdletime: 10
+    Queues:
+      - Name: queue-0
+        CustomActions:
+          - Script: s3://{{ bucket_name }}/post_install.sh
+        ComputeResources:
+          - Name: compute-resource-0
+            InstanceType: {{ instance }}
+        Networking:
+          SubnetIds:
+            - {{ public_subnet_id }}
+        Iam:
```

Comment:
nit: you can use [Yaml anchors](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/) to avoid repeating some blocks.